LICENCE

```diff
@@ -1,5 +1,5 @@
 The MIT License (MIT)
-Copyright (c) 2016 eLife Sciences
+Copyright (c) 2016-2020 eLife Sciences
```

Comment:
do we actually need to do this? In Libero word is that the first version means "from 2016"

Reply:
I can't find anything authoritative on this, but I always took the first date to be the inception or publication date and the second date to be the last revised date so that the copyright term would extend from the last revised date.